homeassistant/strings.json

```diff
@@ -51,6 +51,7 @@
         "already_configured_account": "Account is already configured",
         "already_configured_service": "Service is already configured",
         "already_configured_device": "Device is already configured",
+        "already_in_progress": "Config flow is already in progress.",
```

Comment:
Only string with a "." at the end ? Should we remove this ?

Reply:
Nope, this is not the only one my bad. Maybe we should thin,k about some kind of standardization ?

Reply:
Yup definitely.
```suggestion
        "already_in_progress": "Config flow is already in progress",
```